diff --git a/node_modules/@reactvision/react-viro/components/Material/ViroMaterials.ts b/node_modules/@reactvision/react-viro/components/Material/ViroMaterials.ts
index 977959e..ff92c04 100644
--- a/node_modules/@reactvision/react-viro/components/Material/ViroMaterials.ts
+++ b/node_modules/@reactvision/react-viro/components/Material/ViroMaterials.ts
@@ -17,7 +17,7 @@ import {
   processColor,
 } from "react-native";
 // @ts-ignore
-import assetRegistry from "react-native/Libraries/Image/AssetRegistry";
+import * as assetRegistry from "react-native/Libraries/Image/AssetRegistry";
 // @ts-ignore
 import resolveAssetSource from "react-native/Libraries/Image/resolveAssetSource";
 import { ViroSource } from "../Types/ViroUtils";
diff --git a/node_modules/@reactvision/react-viro/dist/components/Material/ViroMaterials.js b/node_modules/@reactvision/react-viro/dist/components/Material/ViroMaterials.js
index 0494d6c..ef2d479 100644
--- a/node_modules/@reactvision/react-viro/dist/components/Material/ViroMaterials.js
+++ b/node_modules/@reactvision/react-viro/dist/components/Material/ViroMaterials.js
@@ -47,7 +47,7 @@ class ViroMaterials {
                     else {
                         var assetType = "unknown";
                         if (typeof material[prop] !== "object") {
-                            var asset = AssetRegistry_1.default.getAssetByID(material[prop]);
+                            var asset = AssetRegistry_1.getAssetByID(material[prop]);
                             if (asset) {
                                 assetType = asset.type;
                             }
diff --git a/node_modules/@reactvision/react-viro/ios/dist/ViroRenderer/ViroKit.podspec b/node_modules/@reactvision/react-viro/ios/dist/ViroRenderer/ViroKit.podspec
index 7b8e716..db9abc0 100644
--- a/node_modules/@reactvision/react-viro/ios/dist/ViroRenderer/ViroKit.podspec
+++ b/node_modules/@reactvision/react-viro/ios/dist/ViroRenderer/ViroKit.podspec
@@ -10,4 +10,5 @@ Pod::Spec.new do |s|
   s.requires_arc        = true
   s.platform            = :ios, '13.0'
   s.dependency 'React'
+  s.dependency 'PromisesObjC', '~> 2.4'
 end
